api/pkg/apis/v1alpha1/managers/activations/activations-cleanup-manager.go

```diff
@@ -34,16 +32,17 @@ func (s *ActivationsCleanupManager) Init(context *contexts.VendorContext, config
 		return err
 	}
 
-	// Set activation cleanup interval after they are done. If not set, use default 60 minutes.
-	if val, ok := config.Properties["RetentionInMinutes"]; ok {
-		s.RetentionInMinutes, err = strconv.Atoi(val)
+	// Set activation cleanup interval after they are done. If not set, use default 180 days.
+	if val, ok := config.Properties["RetentionDuration"]; ok {
+		s.RetentionDuration, err = time.ParseDuration(val)
 		if err != nil {
-			s.RetentionInMinutes = DefaultRetentionInMinutes
+			s.RetentionDuration = DefaultRetentionDuration
```

Comment:
What if this is a negative time? And I take a deeper look at the supported unit of time.ParseDuration. It cannot support "d" as unit. We need either use 4320h as default value or customize time.ParseDuration to support 180d.

Reply:
Fixed. Will throw exception when duration cannot be parsed or negative.